biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
There's a real bug here. Let me try to explain again from the start:

You have a bundle with two packages: A and B
1) release version 1.0.0 of bundle
2) make a MINOR change in package A
3) remove package B
4) baseline the bundle against version 1.0.0

What would you expect the suggested version to be?

The actual suggested version today is 1.0.1
NOT 1.1.0, or even  2.0.0, it's 1.0.1.

Since we use baseline in CI, this failure to capture the real level of change is problematic.